.travis.yml

```diff
@@ -15,7 +15,7 @@ addons:
 before_install:
   - git clone git://github.com/ladislav-zezula/StormLib.git
   - cd StormLib
-  - cmake CMakeLists.txt
+  - cmake CMakeLists.txt -DBUILD_SHARED_LIBS=ON
```

Comment:
Was it not being built as a shared lib before? (Also, this change might be a bit off topic for the PR...)